### PR TITLE
multiproxy fixes

### PR DESCRIPTION
**Pull request by usmanong**

### CODE REVIEW

File: dfc/daemon.go [hunk 1]
```diff
@@ -294,12 +294,22 @@ func (m *lbmap) versionL() int64 {
 
 func (m *lbmap) cloneU() *lbmap {
 	dst := &lbmap{}
+	m.deepcopy(dst)
```
Comment (usmanong) [inline dfc/daemon.go]: why not let deep copy returns  lbmap? btw, it should be deepCopy()

File: dfc/tests/proxy_test.go [hunk 2]
```diff
@@ -53,7 +53,7 @@ var (
 //       but who knows how many such things exist.
 
 func TestProxy(t *testing.T) {
-	t.Skip("Multi proxy is not ready")
+	// t.Skip("Multi proxy is not ready")
```
Comment (usmanong) [inline dfc/tests/proxy_test.go]: why not delete it?

File: dfc/vote.go [hunk 3]
```diff
@@ -492,6 +506,15 @@ func (h *httprunner) sendElectionRequest(vr *VoteInitiation, nextPrimaryProxy *p
 
 	_, err, _, _ = h.call(&nextPrimaryProxy.daemonInfo, url, http.MethodPut, jsbytes)
 	if err != nil {
+		if IsErrConnectionRefused(err) {
+			for i := 0; i < 5; i++ {
+				time.Sleep(time.Second)
```
Comment (usmanong) [inline dfc/vote.go]: more hacking?

File: dfc/vote.go [hunk 4]
```diff
@@ -31,6 +31,7 @@ type (
 		Candidate string    `json:"candidate"`
 		Primary   string    `json:"primary"`
 		Smap      Smap      `json:"smap"`
+		lbmap     lbmap     `json:"lbmap"`
```
Comment (VladimirMarkelov) [inline dfc/vote.go]: lowcase names are skipped while marshal/unmarshal

**Pull request merged**